ostorlab_insecure_flutter_app/lib/bugs/oracle_padding.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+import 'package:encrypt/encrypt.dart';
+import 'package:ostorlab_insecure_flutter_app/bug_rule.dart';
+
+
+abstract class BugRule {
```

Comment:
Rule not added to the main.

Reply:
Added. [513ff50](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/19/commits/513ff509e5103fdd76e4514316513272325082c9)